config/config.go

```diff
@@ -41,6 +42,11 @@ var (
 	InitConfigCmd = false
 )
 
+// TODO: Placing DiceDB scoped errors here, unless we have a better place to put it
+var (
+	ErrAborted = errors.New("server received ABORT command")
```

Comment:
Lets put this in the errors package. Later we will refactor errors package fully.